Scrapping.py

```diff
@@ -0,0 +1,43 @@
+from bs4 import BeautifulSoup
+import requests
+import pandas as pd
+username=input('Enter Username')
+page=requests.get("https://www.codechef.com/users/"+username)
+soup = BeautifulSoup(page.content, 'html.parser')
+#print(list(soup.children)[2])
+x=str(soup.select('script[type="text/javascript"]')[29]).split(';')[9][19:].strip()
```

Comment:
I get IndexError: list index out of range when running this code. Don't hardcode this.

Reply:
ok ill try to fix it